sys/include/div.h

```diff
@@ -33,17 +33,42 @@ extern "C" {
 /**
  * @brief Approximation of (2**l)/d for d=15625, l=12, 32 bits
  */
-#define DIV_H_INV_15625_32    0x431bde83ul
+#define DIV_H_INV_15625_32     0x431bde83ul
```

Comment:
OK but that means also to change previous code by @gebart. I suppose I should change both?